### PR TITLE
bring button events from iframe to sim

### PR DESCRIPTION
**Pull request by tballmsft**

### CODE REVIEW

**Review by eanders-ms — APPROVED** [review]
Looks good! Apologies, I created some merge conflicts for you. Bad timing on my part.

File: sim/src/hooks/useShieldService.ts [hunk 1]
```diff
@@ -100,7 +100,7 @@ export function useShieldService(canvasRef: HTMLCanvasElement | null) {
 
             switch (msg.type) {
                 case "show-image":
-                    return handleShowImageMessage(msg as protocol.ShowImageMessage)
+                    return handleShowImageMessage(msg as protocol.p.ShowImageMessage)
```
Comment (eanders-ms) [inline sim/src/hooks/useShieldService.ts:103]: Was this change intentional?

Comment (tballmsft) [reply sim/src/hooks/useShieldService.ts:103]: oops
